aries-site/src/pages/components/anchor.mdx

```diff
@@ -115,3 +116,14 @@ Used to indicate that an anchor cannot be interacted with.
 >
   <AnchorDisabledExample />
 </Example>
+
+### Anchor with weight
+
+A weight can be applied to Anchor in cases where extra emphasis is needed. This example demonstrates an Anchor with weight="bold".
```

Comment:
Should we style this as code? I think we have an issue in the backlog to style inline code references, but we can do opportunistically when we come across these instances. Thoughts?

```suggestion
A weight can be applied to Anchor in cases where extra emphasis is needed. This example demonstrates an Anchor with `weight="bold"`.
```